exercises/timesTable13.js

```diff
@@ -8,10 +8,18 @@
  *   13 times 3 is 39
  *   13 times 4 is 52
  */
+//I couldnt figure out how to print out all the multiples under the parameter
```

Comment:
You tried to solve a harder problem! This will print all multiples of 13 smaller than `n`, but I only asked for you to print the first `n` multiples of 13. 😄